test/integration/fleetscope/fleetscope_test.go

```diff
@@ -51,13 +54,110 @@ func TestFleetscope(t *testing.T) {
 				"fleet_project_id":       multitenant.GetStringOutput("fleet_project_id"),
 				"cluster_membership_ids": multitenant.GetStringOutputList("cluster_membership_ids"),
 			}
+
 			fleetscope := tft.NewTFBlueprintTest(t,
 				tft.WithTFDir(fmt.Sprintf("../../../4-fleetscope/envs/%s", envName)),
 				tft.WithVars(vars),
 				tft.WithRetryableTerraformErrors(testutils.RetryableTransientErrors, 3, 2*time.Minute),
 				tft.WithBackendConfig(backendConfig),
 			)
 
+			fleetscope.DefineVerify(func(assert *assert.Assertions) {
+				fleetscope.DefaultVerify(assert)
+
+				// Multitenant Outputs
+				fleetProjectID := multitenant.GetStringOutput("fleet_project_id")
+				clusterRegions := multitenant.GetStringOutputList("cluster_regions")
+				clusterIds := multitenant.GetStringOutputList("clusters_ids")
+				clusterProjectID := multitenant.GetStringOutput("cluster_project_id")
+
+				// Service Account
+				rootReconcilerRoles := []string{"roles/source.reader"}
+				rootReconcilerSa := fmt.Sprintf("root-reconciler@%s.iam.gserviceaccount.com", fleetProjectID)
+				iamReconcilerFilter := fmt.Sprintf("bindings.members:'serviceAccount:%s'", rootReconcilerSa)
+				iamReconcilerCommonArgs := gcloud.WithCommonArgs([]string{"--flatten", "bindings", "--filter", iamReconcilerFilter, "--format", "json"})
+				projectPolicyOp := gcloud.Run(t, fmt.Sprintf("projects get-iam-policy %s", fleetProjectID), iamReconcilerCommonArgs).Array()
+				saReconcilerListRoles := testutils.GetResultFieldStrSlice(projectPolicyOp, "bindings.role")
+				assert.Subset(saReconcilerListRoles, rootReconcilerRoles, fmt.Sprintf("service account %s should have \"roles/source.reader\" project level role", rootReconcilerSa))
+
+				svcRoles := []string{"roles/iam.workloadIdentityUser"}
+				svcSa := fmt.Sprintf("%s.svc.id.goog[config-management-system/root-reconciler]", fleetProjectID)
+				iamSvcFilter := fmt.Sprintf("bindings.members:serviceAccount:'%s'", svcSa)
+				iamSvcCommonArgs := gcloud.WithCommonArgs([]string{"--flatten", "bindings", "--filter", iamSvcFilter, "--format", "json"})
+				svcPolicyOp := gcloud.Run(t, fmt.Sprintf("iam service-accounts get-iam-policy %s --project %s", rootReconcilerSa, fleetProjectID), iamSvcCommonArgs).Array()
+				saSvcListRoles := testutils.GetResultFieldStrSlice(svcPolicyOp, "bindings.role")
+				assert.Subset(saSvcListRoles, svcRoles, fmt.Sprintf("service account %s should have \"roles/iam.workloadIdentityUser\" project level role", svcSa))
+
+				// GKE Feature
+				for _, feature := range []string{
+					"configmanagement",
+					"servicemesh",
+					"multiclusteringress",
+					"multiclusterservicediscovery",
+				} {
+					gkeFeatureOp := gcloud.Runf(t, "container hub features describe %s --project %s", feature, fleetProjectID)
+					assert.Equal("ACTIVE", gkeFeatureOp.Get("resourceState.state").String(), fmt.Sprintf("Hub Feature %s should have resource state equal to ACTIVE", feature))
+
+					if feature == "servicemesh" {
```

Comment:
In the future might want to consider using `switch` `case` as `feature` can only ever match one of these `if` statements.

Reply:
Good call... Will do that!